header footerレスポンシブ作成

tomoharutsutsumi opened the pull request.

ms2sato commented:
@tomoharutsutsumi これ実行する前に、マイグレーションやり直すとエラーになるので、まずそれを修正してください。

```
(64_make_header_footer_responsive) $ bin/rake db:migrate:reset
Running via Spring preloader in process 21976
Dropped database 'db/development.sqlite3'
Dropped database 'db/test.sqlite3'
Created database 'db/development.sqlite3'
Created database 'db/test.sqlite3'
rake aborted!
ActiveRecord::DuplicateMigrationNameError: 

Multiple migrations have the name ActsAsTaggableOnMigration.

/Users/ms2/dev/work/training/tsutsumi/Comilog_Revised_Edition/vendor/bundle/ruby/2.3.0/gems/activerecord-5.1.6/lib/active_record/migration.rb:1254:in `validate'
/Users/ms2/dev/work/training/tsutsumi/Comilog_Revised_Edition/vendor/bundle/ruby/2.3.0/gems/activerecord-5.1.6/lib/active_record/migration.rb:1123:in `initialize'
/Users/ms2/dev/work/training/tsutsumi/Comilog_Revised_Edition/vendor/bundle/ruby/2.3.0/gems/activerecord-5.1.6/lib/active_record/migration.rb:1007:in `new'
/Users/ms2/dev/work/training/tsutsumi/Comilog_Revised_Edition/vendor/bundle/ruby/2.3.0/gems/activerecord-5.1.6/lib/active_record/migration.rb:1007:in `up'
/Users/ms2/dev/work/training/tsutsumi/Comilog_Revised_Edition/vendor/bundle/ruby/2.3.0/gems/activerecord-5.1.6/lib/active_record/migration.rb:985:in `migrate'
/Users/ms2/dev/work/training/tsutsumi/Comilog_Revised_Edition/vendor/bundle/ruby/2.3.0/gems/activerecord-5.1.6/lib/active_record/tasks/database_tasks.rb:171:in `migrate'
/Users/ms2/dev/work/training/tsutsumi/Comilog_Revised_Edition/vendor/bundle/ruby/2.3.0/gems/activerecord-5.1.6/lib/active_record/railties/databases.rake:58:in `block (2 levels) in <top (required)>'
/Users/ms2/dev/work/training/tsutsumi/Comilog_Revised_Edition/vendor/bundle/ruby/2.3.0/gems/activesupport-5.1.6/lib/active_support/dependencies.rb:286:in `load'
/Users/ms2/dev/work/training/tsutsumi/Comilog_Revised_Edition/vendor/bundle/ruby/2.3.0/gems/activesupport-5.1.6/lib/active_support/dependencies.rb:286:in `block in load'
/Users/ms2/dev/work/training/tsutsumi/Comilog_Revised_Edition/vendor/bundle/ruby/2.3.0/gems/activesupport-5.1.6/lib/active_support/dependencies.rb:258:in `load_dependency'
/Users/ms2/dev/work/training/tsutsumi/Comilog_Revised_Edition/vendor/bundle/ruby/2.3.0/gems/activesupport-5.1.6/lib/active_support/dependencies.rb:286:in `load'
-e:1:in `<main>'
Tasks: TOP => db:migrate:reset => db:migrate
(See full trace by running task with --trace)
```

tomoharutsutsumi commented:
@ms2sato 

今確かめてみたのですが、自分のところでは問題なく動いていて。。。。。（masterでも74でも試してみました。）
佐藤さんのエラーを見るとsqliteですが、こちらではpgです。。。
gem pgはmasterにマージされているのですが。。。　こちらのDBの変更を反映させるという動きでよろしいでしょうか。。。。

何卒よろしくお願いします。

```
tomoharu-no-MacBook-Air:Comilog_Revised_Edition tomoharutsutsumi$ bin/rake db:migrate:reset
Running via Spring preloader in process 13432
Dropped database 'comilog_db_revised'
Dropped database 'comilog_db_revised'
Created database 'comilog_db_revised'
Database 'comilog_db_revised' already exists
== 20180503050205 CreateComics: migrating =====================================
-- create_table(:comics)
   -> 0.0233s
== 20180503050205 CreateComics: migrated (0.0235s) ============================

== 20180503051737 DeviseCreateUsers: migrating ================================
-- create_table(:users)
   -> 0.0145s
-- add_index(:users, :email, {:unique=>true})
   -> 0.0196s
-- add_index(:users, :reset_password_token, {:unique=>true})
   -> 0.0094s
-- add_index(:users, :confirmation_token, {:unique=>true})
   -> 0.0114s
-- add_index(:users, :unlock_token, {:unique=>true})
   -> 0.0114s
== 20180503051737 DeviseCreateUsers: migrated (0.0670s) =======================

== 20180503054524 AddColumnToUsers: migrating =================================
-- add_column(:users, :circlename, :string)
   -> 0.0016s
-- add_column(:users, :authorname, :string)
   -> 0.0007s
-- add_column(:users, :username, :string)
   -> 0.0008s
-- add_column(:users, :image, :string)
   -> 0.0010s
-- add_column(:users, :nickname, :string)
   -> 0.0014s
-- add_column(:users, :provider, :string)
   -> 0.0009s
-- add_column(:users, :uid, :string)
   -> 0.0008s
== 20180503054524 AddColumnToUsers: migrated (0.0078s) ========================

== 20180504061012 CreateLikes: migrating ======================================
-- create_table(:likes)
   -> 0.0119s
== 20180504061012 CreateLikes: migrated (0.0122s) =============================

== 20180504065704 AddUserIdToComics: migrating ================================
-- add_column(:comics, :user_id, :integer)
   -> 0.0012s
== 20180504065704 AddUserIdToComics: migrated (0.0013s) =======================

== 20180516112112 AddStatusToComics: migrating ================================
-- add_column(:comics, :status, :integer, {:default=>0, :null=>false, :limit=>1})
   -> 0.0177s
== 20180516112112 AddStatusToComics: migrated (0.0179s) =======================

== 20180519020803 ChangeStatusOfComics: migrating =============================
-- change_column(:comics, :status, :integer, {:default=>1, :null=>false, :limit=>1})
   -> 0.0154s
== 20180519020803 ChangeStatusOfComics: migrated (0.0155s) ====================

== 20180519053030 AddLikesCountToComics: migrating ============================
-- add_column(:comics, :likes_count, :integer, {:default=>0})
   -> 0.0298s
== 20180519053030 AddLikesCountToComics: migrated (0.0299s) ===================

== 20180524142144 RenameTypeColumnToComics: migrating =========================
-- rename_column(:comics, :type, :category)
   -> 0.0030s
== 20180524142144 RenameTypeColumnToComics: migrated (0.0031s) ================

== 20180611154440 ActsAsTaggableOnMigration: migrating ========================
-- adapter_name()
   -> 0.0000s
-- adapter_name()
   -> 0.0000s
-- create_table(:tags, {:id=>:integer})
   -> 0.0100s
-- adapter_name()
   -> 0.0000s
-- adapter_name()
   -> 0.0000s
-- create_table(:taggings, {:id=>:integer})
   -> 0.0120s
-- add_index(:taggings, :tag_id)
   -> 0.0117s
-- add_index(:taggings, [:taggable_id, :taggable_type, :context])
   -> 0.0044s
== 20180611154440 ActsAsTaggableOnMigration: migrated (0.0387s) ===============

== 20180611154441 AddMissingUniqueIndices: migrating ==========================
-- add_index(:tags, :name, {:unique=>true})
   -> 0.0041s
-- index_name(:taggings, {:column=>["tag_id"]})
   -> 0.0000s
-- index_exists?(:taggings, :tag_id, {:name=>"index_taggings_on_tag_id"})
   -> 0.0037s
-- index_name(:taggings, {:column=>:tag_id})
   -> 0.0000s
-- index_name_exists?(:taggings, "index_taggings_on_tag_id")
   -> 0.0014s
-- remove_index(:taggings, {:column=>:tag_id, :name=>"index_taggings_on_tag_id"})
   -> 0.0045s
-- index_name(:taggings, {:column=>[:taggable_id, :taggable_type, :context]})
   -> 0.0000s
-- index_name_exists?(:taggings, "index_taggings_on_taggable_id_and_taggable_type_and_context")
   -> 0.0012s
-- remove_index(:taggings, {:column=>[:taggable_id, :taggable_type, :context], :name=>"index_taggings_on_taggable_id_and_taggable_type_and_context"})
   -> 0.0038s
-- add_index(:taggings, [:tag_id, :taggable_id, :taggable_type, :context, :tagger_id, :tagger_type], {:unique=>true, :name=>"taggings_idx"})
   -> 0.0067s
== 20180611154441 AddMissingUniqueIndices: migrated (0.0267s) =================

== 20180611154442 AddTaggingsCounterCacheToTags: migrating ====================
-- add_column(:tags, :taggings_count, :integer, {:default=>0})
   -> 0.0128s
== 20180611154442 AddTaggingsCounterCacheToTags: migrated (0.0313s) ===========

== 20180611154443 AddMissingTaggableIndex: migrating ==========================
-- add_index(:taggings, [:taggable_id, :taggable_type, :context])
   -> 0.0045s
== 20180611154443 AddMissingTaggableIndex: migrated (0.0049s) =================

== 20180611154444 ChangeCollationForTagNames: migrating =======================
== 20180611154444 ChangeCollationForTagNames: migrated (0.0023s) ==============

== 20180611154445 AddMissingIndexesOnTaggings: migrating ======================
-- index_name(:taggings, {:column=>["tag_id"]})
   -> 0.0002s
-- index_exists?(:taggings, :tag_id, {:name=>"index_taggings_on_tag_id"})
   -> 0.0045s
-- add_index(:taggings, :tag_id)
   -> 0.0099s
-- index_name(:taggings, {:column=>["taggable_id"]})
   -> 0.0000s
-- index_exists?(:taggings, :taggable_id, {:name=>"index_taggings_on_taggable_id"})
   -> 0.0118s
-- add_index(:taggings, :taggable_id)
   -> 0.0042s
-- index_name(:taggings, {:column=>["taggable_type"]})
   -> 0.0000s
-- index_exists?(:taggings, :taggable_type, {:name=>"index_taggings_on_taggable_type"})
   -> 0.0050s
-- add_index(:taggings, :taggable_type)
   -> 0.0053s
-- index_name(:taggings, {:column=>["tagger_id"]})
   -> 0.0000s
-- index_exists?(:taggings, :tagger_id, {:name=>"index_taggings_on_tagger_id"})
   -> 0.0083s
-- add_index(:taggings, :tagger_id)
   -> 0.0046s
-- index_name(:taggings, {:column=>["context"]})
   -> 0.0000s
-- index_exists?(:taggings, :context, {:name=>"index_taggings_on_context"})
   -> 0.0088s
-- add_index(:taggings, :context)
   -> 0.0068s
-- index_name(:taggings, {:column=>["tagger_id", "tagger_type"]})
   -> 0.0000s
-- index_exists?(:taggings, [:tagger_id, :tagger_type], {:name=>"index_taggings_on_tagger_id_and_tagger_type"})
   -> 0.0076s
-- add_index(:taggings, [:tagger_id, :tagger_type])
   -> 0.0051s
-- index_exists?(:taggings, [:taggable_id, :taggable_type, :tagger_id, :context], {:name=>"taggings_idy"})
   -> 0.0085s
-- add_index(:taggings, [:taggable_id, :taggable_type, :tagger_id, :context], {:name=>"taggings_idy"})
   -> 0.0044s
== 20180611154445 AddMissingIndexesOnTaggings: migrated (0.0974s) =============

== 20180712133828 RemoveCategoryFromComics: migrating =========================
-- remove_column(:comics, :category, :string)
   -> 0.0018s
== 20180712133828 RemoveCategoryFromComics: migrated (0.0019s) ================

== 20180712134840 AddCategoryToComics: migrating ==============================
-- add_column(:comics, :category, :integer)
   -> 0.0006s
== 20180712134840 AddCategoryToComics: migrated (0.0006s) =====================



```
